extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/DevServicesElasticsearchProcessor.java

```diff
@@ -49,6 +52,9 @@ public class DevServicesElasticsearchProcessor {
 
     private static final ContainerLocator elasticsearchContainerLocator = new ContainerLocator(DEV_SERVICE_LABEL,
             ELASTICSEARCH_PORT);
+    private static final String DEFAULT_ELASTICSEARCH_IMAGE = "docker.elastic.co/elasticsearch/elasticsearch:8.8.2";
+    private static final String DEFAULT_OPEN_SEARCH_IMAGE = "docker.io/opensearchproject/opensearch:2.9.0";
```

Comment:
FYI there's a way to make sure this will always be the same image as the one in `_attributes.adoc`.

See `io.quarkus.devservices.common.ConfigureUtil#getDefaultImageNameFor` and `extensions/devservices/postgresql/src/main/resources/postgresql-devservice.properties`.